PyMI/src/wmi/__init__.py

```diff
@@ -218,6 +218,24 @@ def get_class_name(self):
     def get_class(self):
         pass
 
+    def __str__(self):
+        try:
+            obj = self.get_wrapped_object()
+
+            obj_string = 'instance if %s\n{' % obj.get_class_name()
+
+            # an MI object's length is the number of attributes the object it
+            # represents has.
+            for i in range(len(obj)):
```

Comment:
So you're using the same format that the old WMI lib had. Should we do something similar for methods as well?

The also have the 'properties' and 'methods' dict fields. Those may be useful as well and you may just use that when building the string representation.